### PR TITLE
brainbuilder -> 0.11.1

### PR DESCRIPTION
**Pull request by arsenius7**
* py-voxcell -> 2.6.0

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-voxcell/package.py [hunk 1]
```diff
@@ -25,3 +26,5 @@ class PyVoxcell(PythonPackage):
     depends_on('py-requests@2.18:', type='run')
     depends_on('py-scipy@0.13:', type='run')
     depends_on('py-six@1.0:', type='run')
+
+    depends_on('libsonata+python@develop', type='run')
```
Comment (arsenius7) [inline var/spack/repos/builtin/packages/py-voxcell/package.py]: to be replaced with `py-libsonata` once / if #350 is merged.

Comment (pramodk) [reply var/spack/repos/builtin/packages/py-voxcell/package.py]: @arsenius7 : ~Currently `libsonata@develop` is not building (afaik).~ Didn't see latest commit.

Comment (arsenius7) [reply var/spack/repos/builtin/packages/py-voxcell/package.py]: > to be replaced with `py-libsonata` once / if #350 is merged.

At #350 we reverted to `libsonata+python` for the moment.

**Review by matz-e — APPROVED** [review]
LGTM to merge if you tested the resulting module from the PR directory.

**Comment by arsenius7** [comment]
> if you tested the resulting module from the PR directory

Thanks for the reminder, I indeed tend to forget about this useful CI feature.
Yes, it works as expected.

**Pull request merged**